bee-hive/poetry.lock

```diff
@@ -0,0 +1,600 @@
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
this file should be removed from the PR (it can help with reproducibility, but causes additional overhead I don't think we need currently - as per team discussion)